hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/connector/HazelcastWriters.java

```diff
@@ -55,6 +67,133 @@
     private HazelcastWriters() {
     }
 
+    @Nonnull
+    @SuppressWarnings("unchecked")
+    public static <T, K, V> ProcessorMetaSupplier mergeMapP(
```

Comment:
This method and the next `updateMapP` are exactly the same, except for the `entryProcessor`. It should be extracted out.